sections/1-introduction.tex

```diff
@@ -104,7 +79,6 @@ \section{Development Model - NABIL}
   \includegraphics[width=45mm]{figures/cumulative_authors.pdf} \\
 (a) & (b) & (c) \\
 \end{tabular}
-\caption{(a) The }
+\caption{(a) We need a caption here. }
```

Comment:
Sorry for this gibberish:

(a) This figure charts the amount of lines of Python code **(dotted black line?)** and total line count **(solid black line?)** with each major release of SunPy **(where is master? is that actually 0.9?)**. It details a steady increase of the SunPy codebase with larger additions to our documentation over the last few major versions of SunPy **(could be the other way around)**. The most striking are the reductions after version 0.9, going into 1.0. This was a period that undersaw major phases of code organization, deletion of obsolete features as well as removing Python 2 support. 

(b) This figure showcases the amount of authors to the number of commits they have done. It should be noted that the number of commits is not a 100% useful measure as any one commit could have as many code or line changes as the previous 100 commits. However, it does indicate (much like Wikipedia) the majority of commits within the package are undertaken by the least amount of people with on average, contributions from an individual is typically less than 10 commits. **(why is the curve there? shouldn't the axes be the other way around?)**  

(c) **(can we not merge figure a and c into one?)** This figure charts the amount of authors (i.e., committers) to the SunPy package as a function of time. The package has seen a steady rise with time as word of mouth and the community around SunPy has developed and grown. Two periods of time stand out, mid-2015 and early 2018, which saw a large increase in the amount of authors over a shorter period than normal **(do we know why?)**.

Reply:
@nabobalis could you move of these comments into issues please? It's not clear how to deal with them here. Thanks.

Reply:
These are captions for the figure (with some questions).

Reply:
@nabobalis do you mind re-writing this (you can paste it into this PR as you had before) with the questions answered? And also slightly more concisely :sunglasses: ?